pkgs/development/python-modules/tensorflow-probability/default.nix

```diff
@@ -11,47 +9,31 @@
 , tensorflow
 , six
 , numpy
+, dm-tree
+, keras
 , decorator
 , cloudpickle
 , gast
 , hypothesis
 , scipy
+, pandas
+, mpmath
 , matplotlib
 , mock
 , pytest
 }:
 
 let
-  version = "0.8.0";
+  version = "0.15.0";
   pname = "tensorflow_probability";
 
   # first build all binaries and generate setup.py using bazel
   bazel-wheel = buildBazelPackage {
-    bazel = bazel_0_26;
-
     name = "${pname}-${version}-py2.py3-none-any.whl";
-
-    src = fetchFromGitHub {
-      owner = "tensorflow";
-      repo = "probability";
-      rev = version;
-      sha256 = "07cm8zba8n0ihzdm3k4a4rsg5v62xxsfvcw4h0niz91c0parqjqy";
+    src = builtins.fetchurl {
```

Comment:
Please use fethFromGitHub and substitute the version.

Reply:
Definitely, thanks. I can't even remember why I used `fetchurl` in the first place.